### PR TITLE
New package: NCHSoftware.WavePad version 19.50

### LINKED ISSUE #172347: [Package Request]: WavePad
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://www.nch.com.au/wavepad/index.html
Publisher: NCHSoftware
Package Name: WavePad
Description: Fully featured professional audio and music editor.
Package Version: 19.50
Installer URL: https://www.nch.com.au/components/wpsetup.exe

### PR DESCRIPTION
**Pull request by Dragon1573**
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #172347

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/172443)

### CODE REVIEW

File: manifests/n/NCHSoftware/WavePad/19.50/NCHSoftware.WavePad.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: NCH.WavePad
```
Comment (Dragon1573) [inline manifests/n/NCHSoftware/WavePad/19.50/NCHSoftware.WavePad.installer.yaml:4]:
```suggestion
PackageIdentifier: NCHSoftware.WavePad
```

File: manifests/n/NCHSoftware/WavePad/19.50/NCHSoftware.WavePad.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: NCH.WavePad
```
Comment (Dragon1573) [inline manifests/n/NCHSoftware/WavePad/19.50/NCHSoftware.WavePad.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: NCHSoftware.WavePad
```

File: manifests/n/NCHSoftware/WavePad/19.50/NCHSoftware.WavePad.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: NCH.WavePad
```
Comment (Dragon1573) [inline manifests/n/NCHSoftware/WavePad/19.50/NCHSoftware.WavePad.yaml:4]:
```suggestion
PackageIdentifier: NCHSoftware.WavePad
```

**Comment by stephengillie** [comment]
Verify:
image
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**